packages/es-components/src/components/containers/notification/useNotification.js

```diff
@@ -65,7 +71,7 @@ function NotificationContent(props) {
 
 const Notification = styled.div`
   align-items: center;
-  background-color: ${props => props.variant.bgColor};
+  background-color: ${props => props.bgColor ?? props.variant.bgColor};
```

Comment:
If `props.bgColor` is null, the background color will use from `props.variant.bgColor`

Reply:
I'm not sure this work is actually necessary, and I think it convolutes the API.

`Notification` is required to be used with a named `type` prop, which is responsible for all the styles in that Notification to correlate with our design system.

If you need a custom Notification, you can always extend it with styled components like this:
```js
const SpecialNotification = styled(Notification)`
  background-color: ${props => props.bgColor};
}`;

// ...later
function SpecialDisplayNotification {
  return (
    <SpecialNotification bgColor="perwinkle">
      This notification is very special!
    </SpecialNotification>
  );
}
```